boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -399,4 +418,157 @@ TEST_F(
     }
 }
 
+// DCM-8
+TEST_F(
+    DcmStateMachineTest,
+    regen_allowed_only_when_going_faster_than_5kph_and_airs_closed)
```

Comment:
You should split this test into a couple of tests. One way to do it would be splitting the code below into 3 tests.

(1) Both airs are opened

(2) One of the airs is opened

(3) Both airs are closed

Reply:
Yeah good point, I updated

Reply:
Nice. Much better to read now